src/main/java/uk/gov/hmcts/reform/hmc/service/HearingManagementServiceImpl.java

```diff
@@ -292,15 +296,31 @@ public HearingResponse deleteHearingRequest(Long hearingId, DeleteHearingRequest
         validateVersionNumber(hearingId, deleteRequest.getVersionNumber());
         validateDeleteHearingStatus(hearingId);
         updateCancellationReasons(hearingId, deleteRequest.getCancellationReasonCode());
-        HearingEntity savedEntity = updateHearingStatus(hearingId);
+        HearingEntity savedEntity = updateHearingStatusAndVersionNumber(hearingId, CANCELLATION_REQUESTED);
         return getSaveHearingResponseDetails(savedEntity);
     }
 
-    private HearingEntity updateHearingStatus(Long hearingId) {
+    private HearingEntity updateHearingStatusAndVersionNumber(Long hearingId, String newStatus) {
         Optional<HearingEntity> hearingResult = hearingRepository.findById(hearingId);
+
         if (hearingResult.isPresent()) {
             final HearingEntity hearingEntity = hearingResult.get();
-            hearingEntity.setStatus(CANCELLATION_REQUESTED);
+            if (null != newStatus) {
+                log.info("CHANGING: Hearing status {}", hearingEntity.getStatus());
+            }
+            log.info("CHANGING: version number : {}", hearingEntity.getCaseHearingRequest().getVersionNumber());
+            if (null != hearingEntity.getCaseHearingRequest()) {
+                hearingEntity.getCaseHearingRequest().setVersionNumber(
+                        hearingEntity.getCaseHearingRequest().getVersionNumber() + 1);
+                log.info("TO: version number : {}",
+                        hearingEntity.getCaseHearingRequest().getVersionNumber());
+                if (null != newStatus) {
+                    hearingEntity.setStatus(newStatus);
```

Comment:
log.info("CHANGING: Hearing status {}", hearingEntity.getStatus()); can be added before setting the status and removed from line 309